### DISCUSSION TITLE
Power consumption

### DISCUSSION THREAD

**Discussion by iospoisson** [amanusk/s-tui#203] [General]
How is defined power consumption w per hours ? Like I have mine showing current power 82w  is this per hour?thanks

**Comment by kuolemaaa** [comment]
Watt (here: power 82w) is not an energy unit. You are confusing the issue.
Watt measures instantaneous power absorbed by the system. You are maybe looking for energy.

Power = Energy / Time
Watt = Joule [J] / Seconds [s]
Watt = WattHour [Wh] / Hour [h]

For example: an hairdryer rated 1000W will absorb 1000W when turned on.
1000W hairdryer turned on for 1 min(60secs) will consume 60000 Joules = 1000Watt x 60 seconds; or 16.66 Wh = 1000Watt x 0.0166 Hours (aka 60 seconds)

Or, again: turned on for one hour at 1000W for every instant of time:
3 600 000 Joules = 1000 Watt x 3600 seconds  OR 1000WattHour = 1000Watt x 1Hour
1000WattHour is commonly referred to "1 kilo-watt-hours" or "1 kWh"
